csharp/src/sdk/Geometry/Color.cs

```diff
@@ -6,7 +6,35 @@ namespace Hypar.Geometry
     /// Color represents an RGBA color.
     /// </summary>
     public class Color
-    {   
+    {
+        public static readonly float[] Aqua = { 0.3f, 0.7f, 0.7f, 0.0f };
```

Comment:
I'd like to do this how `System.Drawing.Color` works in .net. Convert all of these to static properties like:
```c#
public static Color Red
{
  get{ return new Color(1.0f, 0.0f, 0.0f, 0.0f);}
}
```
The difference would be this:
as proposed:
```
var red = new Color(Color.Red);
```
updated:
```
var red = Color.Red
```

Reply:
Ah okay. That's more elegant, thanks.